Sources/TuistKit/Services/Cloud/CloudOrganizationBillingService.swift

```diff
@@ -8,27 +8,37 @@
     protocol CloudOrganizationBillingServicing {
         func run(
             organizationName: String,
-            serverURL: String?
+            directory: String?
         ) async throws
     }
 
     final class CloudOrganizationBillingService: CloudOrganizationBillingServicing {
         private let cloudURLService: CloudURLServicing
         private let opener: Opening
+        private let configLoader: ConfigLoading
 
         init(
             cloudURLService: CloudURLServicing = CloudURLService(),
-            opener: Opening = Opener()
+            opener: Opening = Opener(),
+            configLoader: ConfigLoading = ConfigLoader()
         ) {
             self.cloudURLService = cloudURLService
             self.opener = opener
+            self.configLoader = configLoader
         }
 
         func run(
             organizationName: String,
-            serverURL: String?
+            directory: String?
         ) async throws {
-            let cloudURL = try cloudURLService.url(serverURL: serverURL)
+            let directoryPath: AbsolutePath
+            if let directory {
+                directoryPath = try AbsolutePath(validating: directory, relativeTo: FileHandler.shared.currentPath)
+            } else {
+                directoryPath = FileHandler.shared.currentPath
+            }
+            let config = try configLoader.loadConfig(path: directoryPath)
+            let cloudURL = try cloudURLService.url(configCloudURL: config.cloud?.url)
```

Comment:
What do you think about extracting this code into a common service instead of copy-pasting it in multiple places?

Reply:
Sure! I'd get this out the door and refactor it later though.